ansible_collections/arista/avd/molecule/eos_designs_negative_unit_tests/inventory/host_vars/connected-endpoints-mismatched-descriptions.yml

```diff
@@ -0,0 +1,20 @@
+loopback_ipv4_pool: 192.168.0.0/24
+
+type: l2leaf
+l2leaf:
+  defaults:
+  nodes:
+    - name: connected-endpoints-mismatched-descriptions
+
+servers:
+  # port-channel provide physical and individual port-channel descriptions
+  - name: OLD_SW-1/7
+    adapters:
+      - switches: [connected-endpoints-mismatched-descriptions, connected-endpoints-mismatched-descriptions]
+        switch_ports: [Ethernet9, Ethernet10]
+        descriptions: ["PHYSICAL_PORT_DESCRIPTION_1", "PHYSICAL_PORT_DESCRIPTION_2", "NEGATIVE_TEST_DESCR"]
+        port_channel:
+          mode: "active"
+          description: "PORT_CHANNEL_DESCRIPTION"
+
+expected_error_message: "Length of lists 'switches', 'switch_ports', and 'descriptions' (if used) must match for adapter."
```

Comment:
We should provide some context. Maybe the server name and maybe the connected_endpoints key ?

Reply:
let me know what you think of the change